lib/archethic/transaction_chain.ex

```diff
@@ -1218,4 +1219,16 @@ defmodule Archethic.TransactionChain do
   """
   @spec load_transaction(Transaction.t()) :: :ok
   defdelegate load_transaction(tx), to: MemTablesLoader
+
+  @doc """
+  Return the list inputs for a given transaction
+  """
+  @spec stream_inputs(binary()) :: Enumerable.t() | list(VersionedTransactionInput.t())
+  defdelegate stream_inputs(adddress), to: DBLedger
```

Comment:
To be consistent with the TransactionChain module API (get is from local, fetch is from remote) you should rename the function `get_inputs`.